appeals/web/src/server/appeals/appeal-details/appellant-case/outcome-valid/outcome-valid.controller.js

```diff
@@ -86,11 +86,20 @@ const renderValidDatePage = async (request, response, apiErrors) => {
 	}
 	const { appealId, appealReference } = request.session;
 
-	const mappedPageContent = updateValidDatePage(appealId, appealReference);
-
+	let dateValidDay = request.body['valid-date-day'];
```

Comment:
These should be `const` as there's no later modification of the values - in fact, these variables aren't strictly necessary (you could just pass `request.body['valid-date-day']` etc. directly into the mapper function)

Reply:
👍